### PR TITLE
fix(slider): extract variables for metrics

### PR DESCRIPTION
**Pull request by inikolova**

### CODE REVIEW

File: scss/slider/_layout.scss [hunk 1]
```diff
@@ -259,16 +262,29 @@ $slider-track-thickness: 4px !default;
...
 
+        $slider-draghandle-offset: -(($slider-draghandle-size - ($slider-track-thickness / 2)) / 2) !default;
+        $slider-draghandle-active-offset: -(($slider-draghandle-active-size - ($slider-track-thickness / 2)) / 2) !default;
+
         .k-slider-horizontal & {
-            height: 14px;
-            top: -6px;
-            width: 14px;
+            height: $slider-draghandle-size;
+            top: $slider-draghandle-offset;
```
Comment (joneff) [inline scss/slider/_layout.scss]: Could we go with `top: 50%; transform: translateY(-50%)` ?

Comment (inikolova) [reply scss/slider/_layout.scss]: Yes, we could. Would you like to make this change?

File: scss/slider/_layout.scss [hunk 2]
```diff
@@ -259,16 +262,29 @@ $slider-track-thickness: 4px !default;
...
+            }
         }
 
         .k-slider-vertical & {
-            height: 14px;
-            left: -6px;
-            width: 14px;
+            height: $slider-draghandle-size;
+            left: $slider-draghandle-offset;
+            width: $slider-draghandle-size;
```
Comment (gyoshev) [inline scss/slider/_layout.scss]: seems we have the same width / height declarations for both vertical and horizontal handles, these can be moved to the draghandle in general.

File: scss/slider/_layout.scss (hunk 1 above)
Comment (joneff) [reply scss/slider/_layout.scss]: Sure

**Pull request merged**